legal-api/report-templates/amalgamationApplication.html

```diff
@@ -21,7 +21,15 @@
         </tr>
       </table>
       <div class="business-name-header">
-        <label class="lbl-business-name">{{business.legalName}}</label>
+        <label class="lbl-business-name">
+          {% if business.legalName %}
+               {{ business.legalName }}
+          {% elif nameRequest.legalName %}
+              {{ nameRequest.legalName }}
+          {% else %}
+            {{numberedDescription}}
+          {% endif %}
+        </label>
```

Comment:
Foreign companies will have 'legalName' instead of 'name' in the json (and possibly vice versa), this is to handle those situations.